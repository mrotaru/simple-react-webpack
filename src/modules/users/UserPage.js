import React from 'react'
import { Link } from 'react-router-dom'

const UserPage = ({ match: { params : { userId } } }) => {
  return (
    <>
      <h1>{`User Page: ${userId}`}</h1>
      <Link to='/users'> >>Go to users page</Link>
    </>
  )
}

export default UserPage
