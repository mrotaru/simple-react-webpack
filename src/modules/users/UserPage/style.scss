$my-color: lightblue;

.userDiv {
  background-color: $my-color;
}