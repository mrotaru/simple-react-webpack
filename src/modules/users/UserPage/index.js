import React from 'react'
import { Link } from 'react-router-dom'

import './style.scss'

const UserPage = ({
  match: {
    params: { userId },
  },
}) => {
  return (
    <div class='userDiv'>
      <h1>{`User Page: ${userId}`}</h1>
      <Link to="/users"> >>Go to users page</Link>
    </div>
  )
}

export default UserPage
