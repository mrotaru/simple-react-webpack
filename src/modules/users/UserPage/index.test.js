import React from 'react'

import UserPage from '.'

test('render a label', () => {
  const wrapper = shallow(<UserPage match={{ params: { userId: 'foo' }}} />)
  expect(wrapper).toMatchSnapshot()
})
