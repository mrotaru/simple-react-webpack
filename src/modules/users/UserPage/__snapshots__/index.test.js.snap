// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render a label 1`] = `
<div
  class="userDiv"
>
  <Connect(User)
    userId="foo"
  />
  <Link
    replace={false}
    to="/users"
  >
     &gt;&gt;Go to users page
  </Link>
</div>
`;
